.github/workflows/test.yml

```diff
@@ -10,55 +10,8 @@ jobs:
     strategy:
```

Comment:
This commit is build on top of https://github.com/globalize/globalize/pull/824 most changes come from there